src/marqo/core/constants.py

```diff
@@ -4,3 +4,12 @@
 MARQO_DOC_CHUNKS = 'chunks'
 MARQO_DOC_EMBEDDINGS = 'embeddings'
 MARQO_DOC_ID = '_id'
+
+MARQO_SEARCH_METHOD_TENSOR = 'tensor'
+MARQO_SEARCH_METHOD_LEXICAL = 'lexical'
+
+# For hybrid search
+MARQO_RAW_TENSOR_SCORE = '_raw_tensor_score'
```

Comment:
Better to be consistent with naming and have hybrid in it

Reply:
Done. renamed to MARQO_DOC_HYBRID_TENSOR_SCORE

Reply:
owen prefers just _tensor_score for the actual response key